AsyncDisplayKit/Details/ASCollectionViewFlowLayoutInspector.m

```diff
@@ -8,69 +8,172 @@
 //  of patent rights can be found in the PATENTS file in the same directory.
 //
 
-#import <UIKit/UIKit.h>
-
 #import "ASCollectionViewFlowLayoutInspector.h"
 #import "ASCollectionView.h"
 #import "ASAssert.h"
 #import "ASEqualityHelpers.h"
 
-@implementation ASCollectionViewFlowLayoutInspector {
-  BOOL _delegateImplementsReferenceSizeForHeader;
-  BOOL _delegateImplementsReferenceSizeForFooter;
+#define kDefaultItemSize CGSizeMake(50, 50)
+
+#pragma mark - Helper Functions
+
+// Returns a constrained size to let the cells layout itself as far as possible based on the scrollable direction
+// of the collection view
+static ASSizeRange ASDefaultConstrainedSizeForNodeForCollectionView(ASCollectionView *collectionView) {
```

Comment:
`NodeConstrainedSizeWithCollectionView`  No need for th AS prefix since its private to the implementation.